### ISSUE TITLE
conda downgrades itself in 4.6

### ISSUE THREAD

**Issue by kalefranz** [conda/conda#7816]
```
docker run --rm -it frolvlad/alpine-glibc
wget https://repo.continuum.io/miniconda/Miniconda3-4.5.11-Linux-x86_64.sh -O miniconda.sh
sh miniconda.sh -bfp ~/miniconda
~/miniconda/bin/conda install -y conda-canary::conda
~/miniconda/bin/conda install conda-build --dry-run
```

```
$ docker run --rm -it frolvlad/alpine-glibc
/ # wget https://repo.continuum.io/miniconda/Miniconda3-4.5.11-Linux-x86_64.sh -O miniconda.sh
Connecting to repo.continuum.io (104.16.19.10:443)
miniconda.sh         100% |****************************************************************************************************************************************| 61108k  0:00:00 ETA
/ # sh miniconda.sh -bfp ~/miniconda
PREFIX=/root/miniconda
installing: python-3.7.0-hc3d631a_0 ...
Python 3.7.0
installing: ca-certificates-2018.03.07-0 ...
installing: conda-env-2.6.0-1 ...
installing: libgcc-ng-8.2.0-hdf63c60_1 ...
installing: libstdcxx-ng-8.2.0-hdf63c60_1 ...
installing: libffi-3.2.1-hd88cf55_4 ...
installing: ncurses-6.1-hf484d3e_0 ...
installing: openssl-1.0.2p-h14c3975_0 ...
installing: xz-5.2.4-h14c3975_4 ...
installing: yaml-0.1.7-had09818_2 ...
installing: zlib-1.2.11-ha838bed_2 ...
installing: libedit-3.1.20170329-h6b74fdf_2 ...
installing: readline-7.0-h7b6447c_5 ...
installing: tk-8.6.8-hbc83047_0 ...
installing: sqlite-3.24.0-h84994c4_0 ...
installing: asn1crypto-0.24.0-py37_0 ...
installing: certifi-2018.8.24-py37_1 ...
installing: chardet-3.0.4-py37_1 ...
installing: idna-2.7-py37_0 ...
installing: pycosat-0.6.3-py37h14c3975_0 ...
installing: pycparser-2.18-py37_1 ...
installing: pysocks-1.6.8-py37_0 ...
installing: ruamel_yaml-0.15.46-py37h14c3975_0 ...
installing: six-1.11.0-py37_1 ...
installing: cffi-1.11.5-py37he75722e_1 ...
installing: setuptools-40.2.0-py37_0 ...
installing: cryptography-2.3.1-py37hc365091_0 ...
installing: wheel-0.31.1-py37_0 ...
installing: pip-10.0.1-py37_0 ...
installing: pyopenssl-18.0.0-py37_0 ...
installing: urllib3-1.23-py37_0 ...
installing: requests-2.19.1-py37_0 ...
installing: conda-4.5.11-py37_0 ...
installation finished.
/ # ~/miniconda/bin/conda install -y conda-canary::conda
Solving environment: done

## Package Plan ##

  environment location: /root/miniconda

  added / updated specs:
    - conda-canary::conda


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-4.6.0b0              |           py37_0         1.6 MB  conda-canary

The following packages will be UPDATED:

    conda: 4.5.11-py37_0 --> 4.6.0b0-py37_0 conda-canary


Downloading and Extracting Packages
conda-4.6.0b0        | 1.6 MB    | ############################################################################################################################################## | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
/ # ~/miniconda/bin/conda install conda-build --dry-run
Collecting package metadata: done
Solving environment: done

## Package Plan ##

  environment location: /root/miniconda

  added / updated specs:
    - conda-build


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    beautifulsoup4-4.6.3       |           py37_0         138 KB
    conda-build-3.15.1         |           py37_0         459 KB
    filelock-3.0.8             |           py37_0          13 KB
    glob2-0.6                  |           py37_0          17 KB
    jinja2-2.10                |           py37_0         183 KB
    markupsafe-1.0             |   py37h14c3975_1          25 KB
    patchelf-0.9               |       hf484d3e_2          68 KB
    pkginfo-1.4.2              |           py37_1          39 KB
    psutil-5.4.7               |   py37h14c3975_0         305 KB
    pytz-2018.5                |           py37_0         231 KB
    pyyaml-3.13                |   py37h14c3975_0         177 KB
    tqdm-4.26.0                |   py37h28b3542_0          59 KB
    ------------------------------------------------------------
                                           Total:         1.7 MB

The following NEW packages will be INSTALLED:

  beautifulsoup4     pkgs/main/linux-64::beautifulsoup4-4.6.3-py37_0
  conda-build        pkgs/main/linux-64::conda-build-3.15.1-py37_0
  filelock           pkgs/main/linux-64::filelock-3.0.8-py37_0
  glob2              pkgs/main/linux-64::glob2-0.6-py37_0
  jinja2             pkgs/main/linux-64::jinja2-2.10-py37_0
  markupsafe         pkgs/main/linux-64::markupsafe-1.0-py37h14c3975_1
  patchelf           pkgs/main/linux-64::patchelf-0.9-hf484d3e_2
  pkginfo            pkgs/main/linux-64::pkginfo-1.4.2-py37_1
  psutil             pkgs/main/linux-64::psutil-5.4.7-py37h14c3975_0
  pytz               pkgs/main/linux-64::pytz-2018.5-py37_0
  pyyaml             pkgs/main/linux-64::pyyaml-3.13-py37h14c3975_0
  tqdm               pkgs/main/linux-64::tqdm-4.26.0-py37h28b3542_0

The following packages will be SUPERSEDED by a higher-priority channel:

  conda                  conda-canary::conda-4.6.0b0-py37_0 --> pkgs/main::conda-4.5.11-py37_0

Dry run. Exiting.
```

However there's no downgrade with

```
docker run --rm -it continuumio/miniconda3
conda install -y conda-canary::conda
conda install conda-build --dry-run
```

```
$ docker run --rm -it continuumio/miniconda3
(base) root@3b57ecdc15c8:/# conda update -y conda
Solving environment: done

## Package Plan ##

  environment location: /opt/conda

  added / updated specs:
    - conda


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-4.5.11               |           py36_0         1.0 MB
    openssl-1.0.2p             |       h14c3975_0         3.5 MB
    certifi-2018.8.24          |           py36_1         140 KB
    ------------------------------------------------------------
                                           Total:         4.6 MB

The following packages will be UPDATED:

    certifi: 2018.4.16-py36_0  --> 2018.8.24-py36_1
    conda:   4.5.4-py36_0      --> 4.5.11-py36_0
    openssl: 1.0.2o-h20670df_0 --> 1.0.2p-h14c3975_0


Downloading and Extracting Packages
conda-4.5.11         |  1.0 MB | ################################################################################################################################################ | 100%
openssl-1.0.2p       |  3.5 MB | ################################################################################################################################################ | 100%
certifi-2018.8.24    |  140 KB | ################################################################################################################################################ | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
(base) root@3b57ecdc15c8:/# conda install -y conda-canary::conda
Solving environment: done

## Package Plan ##

  environment location: /opt/conda

  added / updated specs:
    - conda-canary::conda


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-4.6.0b0              |           py36_0         1.6 MB  conda-canary

The following packages will be UPDATED:

    conda: 4.5.11-py36_0 --> 4.6.0b0-py36_0 conda-canary


Downloading and Extracting Packages
conda-4.6.0b0        | 1.6 MB    | ############################################################################################################################################## | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
(base) root@3b57ecdc15c8:/# conda install conda-build --dry-run
Collecting package metadata: done
Solving environment: done

## Package Plan ##

  environment location: /opt/conda

  added / updated specs:
    - conda-build


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    beautifulsoup4-4.6.3       |           py36_0         138 KB
    conda-build-3.15.1         |           py36_0         459 KB
    filelock-3.0.8             |           py36_0          13 KB
    glob2-0.6                  |           py36_0          17 KB
    jinja2-2.10                |           py36_0         184 KB
    markupsafe-1.0             |   py36h14c3975_1          24 KB
    patchelf-0.9               |       hf484d3e_2          68 KB
    pkginfo-1.4.2              |           py36_1          39 KB
    psutil-5.4.6               |   py36h14c3975_0         303 KB
    pytz-2018.5                |           py36_0         232 KB
    pyyaml-3.13                |   py36h14c3975_0         178 KB
    tqdm-4.26.0                |   py36h28b3542_0          59 KB
    ------------------------------------------------------------
                                           Total:         1.7 MB

The following NEW packages will be INSTALLED:

  beautifulsoup4     pkgs/main/linux-64::beautifulsoup4-4.6.3-py36_0
  conda-build        pkgs/main/linux-64::conda-build-3.15.1-py36_0
  filelock           pkgs/main/linux-64::filelock-3.0.8-py36_0
  glob2              pkgs/main/linux-64::glob2-0.6-py36_0
  jinja2             pkgs/main/linux-64::jinja2-2.10-py36_0
  markupsafe         pkgs/main/linux-64::markupsafe-1.0-py36h14c3975_1
  patchelf           pkgs/main/linux-64::patchelf-0.9-hf484d3e_2
  pkginfo            pkgs/main/linux-64::pkginfo-1.4.2-py36_1
  psutil             pkgs/main/linux-64::psutil-5.4.6-py36h14c3975_0
  pytz               pkgs/main/linux-64::pytz-2018.5-py36_0
  pyyaml             pkgs/main/linux-64::pyyaml-3.13-py36h14c3975_0
  tqdm               pkgs/main/linux-64::tqdm-4.26.0-py36h28b3542_0

Dry run. Exiting.

```

**Comment by mbargull** [comment]
> However there's no downgrade with \[...\]

TLDR; if you `conda install -y conda-canary::conda libgcc-ng`, you see the same behavior with `Miniconda 4.5.4`.
You don't see the same behavior with `4.5.4` since that one ships `libgcc-ng 7.2` and newer builds of packages require `libgcc-ng >=7.3`. Since `4.5.11` has a newer `libgcc-ng`, `conda` is swayed to install more recent builds of some packages. For whatever reason, it wants to change itself in that case.

So, this was fixed for `4.5.x` in e11408e32cfeec7bb3a841d3dd907d54f06d9385 from gh-7598. But something else must have changed on `master` early on so that the merge up of that commit in gh-7603 didn't have the same effect. Meaning, even at gh-7603's merge commit, 3827b5f3d76c6904cc83340c9bcb26331348d5f5, this wants to downgrade `conda`.

**Comment by bounlu** [comment]
I still have the same problem in `conda 24.7.1` and it's very dangerous to downgrade conda back to `22`:

```
$ conda update --all
Channels:
 - conda-canary
 - bioconda
 - r
 - conda-forge
 - defaults
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /opt/anaconda3


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    aom-3.3.0                  |       h27087fc_1         2.6 MB  conda-forge
    arrow-cpp-10.0.1           |   h3e2b116_4_cpu          29 KB  conda-forge
    backports.zoneinfo-0.2.1   |  py310hff52083_8           7 KB  conda-forge
    billiard-4.2.0             |  py310h2372a71_0         157 KB  conda-forge
    biscuit-1.2.0.20230130     |       he272189_0         364 KB  bioconda
    boost-1.85.0               |       hb7f781d_2          18 KB  conda-forge
    boost-cpp-1.85.0           |       h44aadfe_2          17 KB  conda-forge
    bowtie2-2.5.4              |       he20e202_2        14.3 MB  bioconda
    celery-5.3.6               |     pyhd8ed1ab_0         498 KB  conda-forge
    charls-2.2.0               |       h9c3ff4c_0         138 KB  conda-forge
    conda-22.9.0               |  py310hff52083_2         972 KB  conda-forge
    conda-build-3.26.1         |  py310h06a4308_0         601 KB
    conda-libmamba-solver-22.8.1|     pyhd8ed1ab_0          36 KB  conda-forge
    curl-7.87.0                |       h6312ad2_0          86 KB  conda-forge
    dask-2024.8.0              |     pyhd8ed1ab_0           7 KB  conda-forge
    dask-core-2024.8.0         |     pyhd8ed1ab_0         860 KB  conda-forge
    dask-expr-1.1.10           |     pyhd8ed1ab_0         181 KB  conda-forge
    dav1d-1.4.3                |       h4ab18f5_0          29 KB  conda-forge
    dbus-1.13.18               |       hb2f20db_0         504 KB
    distributed-2024.8.0       |     pyhd8ed1ab_0         781 KB  conda-forge
    fmt-11.0.1                 |       h434a139_0         190 KB  conda-forge
    git-2.30.2                 | pl5320h24fefe6_1        12.6 MB  conda-forge
    grpc-cpp-1.51.1            |       h3e2b116_0          21 KB  conda-forge
    gst-plugins-base-1.14.1    |       h6a678d5_1         2.2 MB
    gstreamer-1.14.1           |       h5eee18b_1         1.7 MB
    harfbuzz-4.3.0             |       hf52aaf7_2         1.3 MB
    hdf5-1.10.6                |nompi_h7c3c948_1111         3.1 MB  conda-forge
    jack-0.125.0               |    hf484d3e_1001         875 KB  conda-forge
    jpeg-9e                    |       h166bdaf_2         269 KB  conda-forge
    kaleido-core-0.1.0         |       h3644ca4_0        56.3 MB  conda-forge
    krb5-1.20.1                |       hf9c8cef_0         1.3 MB  conda-forge
    libarchive-3.5.2           |       hb890918_3         1.6 MB  conda-forge
    libavif-0.10.1             |       h166bdaf_0          95 KB  conda-forge
    libboost-1.85.0            |       hba137d9_2         2.8 MB  conda-forge
    libboost-devel-1.85.0      |       h00ab1b0_2          40 KB  conda-forge
    libboost-headers-1.85.0    |       ha770c72_2        13.4 MB  conda-forge
    libboost-python-1.85.0     |  py310he6ccd79_2         120 KB  conda-forge
    libboost-python-devel-1.85.0|  py310hb7f781d_2          21 KB  conda-forge
    libclang-14.0.6            |default_h7634d5b_1         130 KB  conda-forge
    libcurl-7.87.0             |       h6312ad2_0         339 KB  conda-forge
    libdav1d7-1.4.3            |       h4ab18f5_0         732 KB  conda-forge
    libgd-2.3.3                |       h6a678d5_3         239 KB
    libjpeg-turbo-2.1.4        |       h166bdaf_0         988 KB  conda-forge
    libllvm16-16.0.6           |       hb3ce162_3        33.7 MB  conda-forge
    libllvm17-17.0.6           |       hb3ce162_1        34.9 MB  conda-forge
    libmamba-0.27.0            |       h0dd8ff0_0         1.5 MB  conda-forge
    libmambapy-0.27.0          |  py310hab0e683_0         342 KB  conda-forge
    libpq-12.15                |       h37d81fd_1         2.4 MB
    libwebp-1.2.4              |       h1daa5a0_1          82 KB  conda-forge
    libwebp-base-1.2.4         |       h166bdaf_0         404 KB  conda-forge
    libxkbcommon-1.7.0         |       h2c5496b_1         579 KB  conda-forge
    libxslt-1.1.39             |       h76b75d6_0         248 KB  conda-forge
    lxml-5.2.2                 |  py310h6a33d3d_0         1.3 MB  conda-forge
    mathjax-3.2.2              |       ha770c72_0         5.0 MB  conda-forge
    mysql-5.7.20               |    hf484d3e_1001        79.2 MB  conda-forge
    mysql-libs-8.0.32          |       h54cf53e_0         1.5 MB  conda-forge
    openjdk-11.0.15            |       hc6918da_0       174.0 MB  conda-forge
    pillow-9.4.0               |  py310h6a678d5_0         730 KB
    psycopg2-2.9.9             |  py310h5eee18b_0         185 KB
    pulseaudio-client-17.0     |       hb77b528_0         740 KB  conda-forge
    pybigwig-0.3.18            |  py310h722e95c_2          81 KB  bioconda
    pyqt-5.15.4                |  py310hd8f1fbe_0         6.1 MB  conda-forge
    pyqt5-sip-12.9.0           |  py310hd8f1fbe_0          85 KB  conda-forge
    python-kaleido-0.1.0       |     pyhd8ed1ab_0          13 KB  conda-forge
    qt-main-5.15.2             |      h110a718_10        53.7 MB
    sip-6.5.1                  |  py310h122e73d_2         373 KB  conda-forge
    tbb4py-2021.12.0           |  py310h0eae7c2_3          91 KB  conda-forge
    tensorboard-data-server-0.7.0|  py310h52d8a92_0         4.7 MB
    ucsc-wigtobigwig-357       |                0         984 KB  bioconda
    xcb-util-0.4.1             |       hb711507_2          19 KB  conda-forge
    xcb-util-image-0.4.0       |       hb711507_2          24 KB  conda-forge
    xcb-util-keysyms-0.4.1     |       hb711507_0          14 KB  conda-forge
    xcb-util-renderutil-0.3.10 |       hb711507_0          17 KB  conda-forge
    xcb-util-wm-0.4.2          |       hb711507_0          50 KB  conda-forge
    xkeyboard-config-2.42      |       h4ab18f5_0         380 KB  conda-forge
    ------------------------------------------------------------
                                           Total:       525.4 MB

The following NEW packages will be INSTALLED:

  backports.zoneinfo conda-forge/linux-64::backports.zoneinfo-0.2.1-py310hff52083_8 
  jpeg               conda-forge/linux-64::jpeg-9e-h166bdaf_2 
  libboost           conda-forge/linux-64::libboost-1.85.0-hba137d9_2 
  libboost-devel     conda-forge/linux-64::libboost-devel-1.85.0-h00ab1b0_2 
  libdav1d7          conda-forge/linux-64::libdav1d7-1.4.3-h4ab18f5_0 
  mysql              conda-forge/linux-64::mysql-5.7.20-hf484d3e_1001 

The following packages will be UPDATED:

  billiard                          3.6.4.0-py310h5764c6d_3 --> 4.2.0-py310h2372a71_0 
  boost                                   1.82.0-h17c5347_3 --> 1.85.0-hb7f781d_2 
  boost-cpp                               1.82.0-he19a28c_1 --> 1.85.0-h44aadfe_2 
  bowtie                              1.3.1-py310h7b97f60_7 --> 1.3.1-py310h4971c18_9 
  bowtie2                             2.5.3-py310ha0a81b8_1 --> 2.5.4-he20e202_2 
  c-blosc2                                2.12.0-hb4ffafa_0 --> 2.15.1-hc57e6cf_0 
  celery                                 5.2.7-pyhd8ed1ab_0 --> 5.3.6-pyhd8ed1ab_0 
  click              conda-forge/linux-64::click-8.0.4-py3~ --> conda-forge/noarch::click-8.1.7-unix_pyh707e725_0 
  dask                               2023.10.1-pyhd8ed1ab_0 --> 2024.8.0-pyhd8ed1ab_0 
  dask-core                          2023.10.1-pyhd8ed1ab_0 --> 2024.8.0-pyhd8ed1ab_0 
  dask-expr                              0.3.3-pyhd8ed1ab_0 --> 1.1.10-pyhd8ed1ab_0 
  dav1d                                    1.2.1-hd590300_0 --> 1.4.3-h4ab18f5_0 
  dbus                  conda-forge::dbus-1.13.6-h5008d03_3 --> pkgs/main::dbus-1.13.18-hb2f20db_0 
  distributed                        2023.10.1-pyhd8ed1ab_0 --> 2024.8.0-pyhd8ed1ab_0 
  flask                                  2.2.5-pyhd8ed1ab_0 --> 3.0.3-pyhd8ed1ab_0 
  fmt                                     10.2.1-h00ab1b0_0 --> 11.0.1-h434a139_0 
  glib                                    2.78.4-hfc55251_0 --> 2.80.2-hf974151_0 
  glib-tools                              2.78.4-hfc55251_0 --> 2.80.2-hb6ce0ca_0 
  hdf5                            1.10.6-nompi_h3c11f04_101 --> 1.10.6-nompi_h7c3c948_1111 
  huggingface_hub                       0.16.4-pyhd8ed1ab_0 --> 0.24.5-pyhd8ed1ab_0 
  icu                                       72.1-hcb278e6_0 --> 73.2-h59595ed_0 
  libboost-headers                        1.82.0-ha770c72_3 --> 1.85.0-ha770c72_2 
  libboost-python                    1.82.0-py310hcb52e73_3 --> 1.85.0-py310he6ccd79_2 
  libboost-python-d~                 1.82.0-py310h17c5347_3 --> 1.85.0-py310hb7f781d_2 
  libglib                                 2.78.4-h783c2da_0 --> 2.80.2-hf974151_0 
  libhwloc                      2.9.3-default_h554bfaf_1009 --> 2.11.1-default_hecaa2ac_1000 
  libllvm16                               16.0.6-h5cf9203_2 --> 16.0.6-hb3ce162_3 
  libllvm17          pkgs/main::libllvm17-17.0.6-hc9c083f_0 --> conda-forge::libllvm17-17.0.6-hb3ce162_1 
  libxcb                                 1.13-h7f98852_1004 --> 1.16-hd590300_0 
  libxkbcommon                             1.5.0-h79f4944_1 --> 1.7.0-h2c5496b_1 
  libxml2                                 2.11.5-h0d562d8_0 --> 2.12.7-hc051c1a_1 
  libxslt                                 1.1.37-h0054252_1 --> 1.1.39-h76b75d6_0 
  lxml                                4.9.3-py310h9b7343a_1 --> 5.2.2-py310h6a33d3d_0 
  mathjax                                  2.7.7-ha770c72_3 --> 3.2.2-ha770c72_0 
  pcre2                                    10.42-hcad00b1_0 --> 10.43-hcad00b1_0 
  psycopg2           conda-forge::psycopg2-2.9.6-py310h76c~ --> pkgs/main::psycopg2-2.9.9-py310h5eee18b_0 
  pulseaudio-client                         16.1-hb77b528_5 --> 17.0-hb77b528_0 
  python-xxhash                       3.0.0-py310h5764c6d_2 --> 3.4.1-py310h2372a71_0 
  python-zlib-ng                      0.4.3-py310ha5dcb8a_0 --> 0.4.3-py310h6420a16_2 
  tbb                                  2021.11.0-h00ab1b0_1 --> 2021.12.0-h434a139_3 
  tbb4py                          2021.11.0-py310hf5d52d3_1 --> 2021.12.0-py310h0eae7c2_3 
  transformers                                   2.1.1-py_0 --> 4.33.3-pyhd8ed1ab_0 
  xcb-util                                 0.4.0-h516909a_0 --> 0.4.1-hb711507_2 
  xcb-util-image                           0.4.0-h166bdaf_0 --> 0.4.0-hb711507_2 
  xcb-util-keysyms                         0.4.0-h516909a_0 --> 0.4.1-hb711507_0 
  xcb-util-renderut~                       0.3.9-h166bdaf_0 --> 0.3.10-hb711507_0 
  xcb-util-wm                              0.4.1-h516909a_0 --> 0.4.2-hb711507_0 
  xkeyboard-config                          2.38-h0b41bf4_0 --> 2.42-h4ab18f5_0 
  xorg-libx11                              1.8.4-h0b41bf4_0 --> 1.8.9-hb711507_1 
  xorg-libxi                              1.7.10-h7f98852_0 --> 1.7.10-h4bc722e_1 
  xorg-libxrender                      0.9.10-h7f98852_1003 --> 0.9.11-hd590300_0 
  xorg-libxtst                          1.2.3-h7f98852_1002 --> 1.2.5-h4bc722e_0 
  xxhash                                   0.8.0-h7f98852_3 --> 0.8.2-hd590300_0 
  zlib-ng                                  2.0.7-h0b41bf4_0 --> 2.2.1-he02047a_0 

The following packages will be SUPERSEDED by a higher-priority channel:

  cairo              conda-forge::cairo-1.16.0-h35add3b_10~ --> pkgs/main::cairo-1.16.0-hb05425b_5 
  conda-build        conda-forge::conda-build-24.7.1-py310~ --> pkgs/main::conda-build-3.26.1-py310h06a4308_0 
  curl                     pkgs/main::curl-8.7.1-hdbd6064_0 --> conda-forge::curl-7.87.0-h6312ad2_0 
  git                pkgs/main::git-2.45.2-pl5340h9abc3c3_0 --> conda-forge::git-2.30.2-pl5320h24fefe6_1 
  gst-plugins-base   conda-forge::gst-plugins-base-1.22.0-~ --> pkgs/main::gst-plugins-base-1.14.1-h6a678d5_1 
  gstreamer          conda-forge::gstreamer-1.22.0-h25f0c4~ --> pkgs/main::gstreamer-1.14.1-h5eee18b_1 
  harfbuzz           conda-forge::harfbuzz-6.0.0-h3ff4399_1 --> pkgs/main::harfbuzz-4.3.0-hf52aaf7_2 
  imagecodecs        conda-forge::imagecodecs-2023.8.12-py~ --> pkgs/main::imagecodecs-2021.8.26-py310h46e8fbd_2 
  libcurl               pkgs/main::libcurl-8.7.1-h251f7ec_0 --> conda-forge::libcurl-7.87.0-h6312ad2_0 
  libgd                 conda-forge::libgd-2.3.3-hfa28ad5_6 --> pkgs/main::libgd-2.3.3-h6a678d5_3 
  libpq                  conda-forge::libpq-15.3-hbcd7760_1 --> pkgs/main::libpq-12.15-h37d81fd_1 
  libtiff             conda-forge::libtiff-4.5.1-h8b53f26_1 --> pkgs/main::libtiff-4.5.1-h6a678d5_0 
  perl-net-ssleay    conda-forge::perl-net-ssleay-1.92-pl5~ --> bioconda::perl-net-ssleay-1.92-pl5321h0e0aaa8_1 
  pillow             conda-forge::pillow-9.5.0-py310h065c6~ --> pkgs/main::pillow-9.4.0-py310h6a678d5_0 
  qt-main            conda-forge::qt-main-5.15.8-h5c52f38_~ --> pkgs/main::qt-main-5.15.2-h110a718_10 
  tensorboard-data-~ conda-forge::tensorboard-data-server-~ --> pkgs/main::tensorboard-data-server-0.7.0-py310h52d8a92_0 

The following packages will be DOWNGRADED:

  alsa-lib                                 1.2.8-h166bdaf_0 --> 1.2.6.1-h7f98852_0 
  anaconda-anon-usa~                0.4.4-py310hfc0e8ea_100 --> 0.4.4-py310hc06175d_0 
  aom                                      3.5.0-h27087fc_0 --> 3.3.0-h27087fc_1 
  arrow-cpp                           11.0.0-ha770c72_9_cpu --> 10.0.1-h3e2b116_4_cpu 
  aws-c-auth                              0.6.26-h987a71b_2 --> 0.6.21-hd93a3ba_3 
  aws-c-cal                               0.5.21-h48707d8_2 --> 0.5.20-hd3b2fe5_3 
  aws-c-common                            0.8.14-h0b41bf4_0 --> 0.8.5-h166bdaf_0 
  aws-c-compression                       0.2.16-h03acc5a_5 --> 0.2.16-hf5f93bc_0 
  aws-c-event-stream                      0.2.20-h00877a2_4 --> 0.2.17-h57874a7_1 
  aws-c-http                               0.7.6-hf342b9f_0 --> 0.7.0-h96ef541_0 
  aws-c-io                               0.13.19-h5b20300_3 --> 0.13.12-h57ca295_1 
  aws-c-mqtt                              0.8.6-hc4349f7_12 --> 0.7.13-h0b5698f_12 
  aws-c-s3                                 0.2.7-h909e904_1 --> 0.2.1-h5d7e488_4 
  aws-c-sdkutils                           0.1.9-h03acc5a_0 --> 0.1.7-hf5f93bc_0 
  aws-checksums                           0.1.14-h03acc5a_5 --> 0.1.14-h6027aba_0 
  aws-crt-cpp                            0.19.8-hf7fbfca_12 --> 0.18.16-h26430d7_8 
  aws-sdk-cpp                            1.10.57-h17c43bd_8 --> 1.9.379-h56135f1_7 
  biscuit                         1.5.0.20240506-h0be9327_0 --> 1.2.0.20230130-he272189_0 
  charls                                   2.4.2-h59595ed_0 --> 2.2.0-h9c3ff4c_0 
  conda                              24.7.1-py310hff52083_0 --> 22.9.0-py310hff52083_2 
  conda-libmamba-so~                   23.12.0-pyhd8ed1ab_0 --> 22.8.1-pyhd8ed1ab_0 
  cryptography                       42.0.2-py310hb8475ec_0 --> 39.0.0-py310h65dfdc0_0 
  cyvcf2                            0.30.22-py310hcf1fb4a_0 --> 0.30.16-py310h114fdab_1 
  fastp                                   0.23.4-hadf994f_3 --> 0.23.4-h5f740d0_0 
  grpc-cpp                                1.51.1-h27aab58_3 --> 1.51.1-h3e2b116_0 
  grpcio                             1.51.1-py310heca2aa9_3 --> 1.51.1-py310h5b20944_0 
  htslib                                    1.18-h81da01d_0 --> 1.17-h6bc39ce_1 
  jack                                    1.9.22-h11f4161_0 --> 0.125.0-hf484d3e_1001 
  kaleido-core                             0.2.1-h3644ca4_0 --> 0.1.0-h3644ca4_0 
  krb5                                    1.20.1-h81ceb04_0 --> 1.20.1-hf9c8cef_0 
  lcms2                                     2.15-haa2dc70_1 --> 2.15-hfd0df8a_0 
  lerc                                     4.0.0-h27087fc_0 --> 3.0-h9c3ff4c_0 
  libabseil                     20230125.0-cxx17_hcb278e6_1 --> 20220623.0-cxx17_h05df665_6 
  libarchive                               3.7.2-h039dbb9_0 --> 3.5.2-hb890918_3 
  libarrow                            11.0.0-h33598ff_9_cpu --> 10.0.1-hee49ebd_4_cpu 
  libavif                                 0.11.1-h8182462_2 --> 0.10.1-h166bdaf_0 
  libclang                       16.0.6-default_hf981a13_11 --> 14.0.6-default_h7634d5b_1 
  libclang13                     16.0.6-default_h9def88c_11 --> 14.0.6-default_h9986a30_1 
  libdeflate                                1.18-h0b41bf4_0 --> 1.17-h0b41bf4_0 
  libevent                                2.1.10-h28343ad_4 --> 2.1.10-h9b69904_4 
  libgoogle-cloud                          2.8.0-h3c06191_0 --> 2.5.0-h5d50b59_1 
  libgrpc                                 1.51.1-hcf146ea_3 --> 1.51.1-h05bd8bd_0 
  libjpeg-turbo                          2.1.5.1-hd590300_1 --> 2.1.4-h166bdaf_0 
  libmamba                                 1.5.3-had39da4_2 --> 0.27.0-h0dd8ff0_0 
  libmambapy                          1.5.3-py310h39ff949_2 --> 0.27.0-py310hab0e683_0 
  libnghttp2                              1.58.0-h47da74e_0 --> 1.51.0-hdcd2b5c_0 
  libssh2                                 1.11.0-h0841786_0 --> 1.10.0-haa6b8db_3 
  libthrift                               0.18.0-h5e4af38_0 --> 0.16.0-h491838f_2 
  libwebp                                  1.3.1-hbf2b3c1_0 --> 1.2.4-h1daa5a0_1 
  libwebp-base                             1.3.1-hd590300_0 --> 1.2.4-h166bdaf_0 
  mosdepth                                 0.3.6-hd299d5a_0 --> 0.3.3-hd299d5a_3 
  mysql-common                            8.0.33-hf1915f5_6 --> 8.0.32-h14678bc_0 
  mysql-libs                              8.0.33-hca2cd23_6 --> 8.0.32-h54cf53e_0 
  openjdk                                 17.0.3-h4335b31_6 --> 11.0.15-hc6918da_0 
  openssl                                  3.1.6-h4ab18f5_0 --> 1.1.1w-hd590300_0 
  pblat                                    2.5.1-h84c94e8_2 --> 2.5.1-h0e0aaa8_1 
  portaudio                               19.6.0-h583fa2b_7 --> 19.6.0-h57a0ea0_5 
  pyarrow                        11.0.0-py310h633f555_9_cpu --> 10.0.1-py310hf2f6370_4_cpu 
  pybigwig                           0.3.22-py310h79000e5_2 --> 0.3.18-py310h722e95c_2 
  pycurl                             7.45.1-py310h60f9ec7_3 --> 7.45.1-py310h2aed498_3 
  pyopenssl                             24.2.1-pyhd8ed1ab_2 --> 23.2.0-pyhd8ed1ab_1 
  pyqt                               5.15.9-py310h04931ad_5 --> 5.15.4-py310hd8f1fbe_0 
  pyqt5-sip                         12.12.2-py310hc6cd4ac_5 --> 12.9.0-py310hd8f1fbe_0 
  pysam                              0.22.0-py310h41dec4a_0 --> 0.21.0-py310hff46b53_0 
  python                         3.10.13-hd12c33a_0_cpython --> 3.10.8-h257c98d_0_cpython 
  python-kaleido                         0.2.1-pyhd8ed1ab_0 --> 0.1.0-pyhd8ed1ab_0 
  re2                                 2023.02.02-hcb278e6_0 --> 2022.06.01-h27087fc_1 
  rsync                                    3.2.7-h70740c4_0 --> 3.1.3-h1245774_1003 
  s2n                                     1.3.41-h3358134_0 --> 1.3.31-hae46d1a_0 
  samtools                                  1.18-h50ea8bc_1 --> 1.18-hd87286a_0 
  sip                                6.7.12-py310hc6cd4ac_0 --> 6.5.1-py310h122e73d_2 
  streamlit                              1.9.0-pyhd8ed1ab_0 --> 1.8.0-pyhd8ed1ab_0 
  tifffile                           2024.7.24-pyhd8ed1ab_0 --> 2021.11.2-pyhd8ed1ab_0 
  tokenizers                         0.15.0-py310h320607d_0 --> 0.13.2-py310h26b42ff_0 
  ucsc-wigtobigwig                           447-h2a80c09_1 --> 357-0 
  wget                                    1.20.3-ha35d2d1_1 --> 1.20.3-ha56f1ee_1 
  yaml-cpp                                 0.8.0-h59595ed_0 --> 0.7.0-h59595ed_3 
  zfp                                      1.0.1-hac33072_1 --> 0.5.5-h9c3ff4c_8 


Proceed ([y]/n)? 

```

How to prevent conda to downgrade itself permanenty?
